packages/react/src/components/Modal/ModalHeader/ModalHeader.tsx

```diff
@@ -0,0 +1,65 @@
+import type { HTMLAttributes } from 'react';
+import React, { forwardRef, useContext } from 'react';
+import cn from 'classnames';
+import { XMarkIcon } from '@navikt/aksel-icons';
+
+import { Heading, Paragraph } from '../../Typography';
+import { Button } from '../../Button';
+import { ModalContext } from '../ModalContext';
+
+import classes from './ModalHeader.module.css';
+
+type ModalHeaderProps = {
+  closeButton?: boolean;
+  title: React.ReactNode;

```

Comment:
Suggest using `children` for title text instead of separate prop as its more semantically correct to have `<Modal.Header subtitle="My subtitle">My title</Modal.Header>`. In addition we already do this `Accordion` and should do for `Card.Header`.